twindle-cli/twitter/utils/tweet-utils.js

```diff
@@ -19,12 +19,14 @@ const {
  * @returns {string}
  */
 const extractTweetId = (tweet_url) =>
-  [...matchAll(tweet_url, /https?:\/\/twitter.com\/[a-zA-Z_]{1,20}\/status\/([0-9]*)/g)][0][1];
+  //[...matchAll(tweet_url, /https?:\/\/twitter.com\/[a-zA-Z_]{1,20}\/status\/([0-9]*)/g)][0][1];
+  tweet_url.split('/')[5]
```

Comment:
Please revert. Won't match any tweet if https:// is missing. The regex above matches it perfectly.

Reply:
so for this solution can be : a regex pattern like : ` /[0-9]+` , to extract tweet ID will be ok without caring about `http` or `https`